bootstrap/config/manager/manager.yaml

```
@@ -57,5 +57,12 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
+        volumeMounts:
+          - mountPath: /tmp/k8s-webhook-server/serving-certs/
+            name: certs
       serviceAccountName: capoa-bootstrap-controller-manager
       terminationGracePeriodSeconds: 10
+      volumes:
+        - name: certs
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)